client/lib/plans/constants.js

```diff
@@ -621,7 +628,13 @@ export const FEATURES_LIST = {
 		),
 		getStoreSlug: () => 'no-adverts/no-adverts.php'
 	},
-
+	[ FEATURE_REPUBLICIZE ]: {
+		getSlug: () => FEATURE_REPUBLICIZE,
+		getTitle: () => i18n.translate( 'Social sharing menu' ),
+		getDescription: () => i18n.translate(
+			'Share any post on connected social media accounts'
```

Comment:
Maybe we should make it more clear that it differs from social sharing embedded in the post editor.